app/src/main/java/com/x8bit/bitwarden/ui/platform/components/card/BitwardenActionCard.kt

```diff
@@ -57,12 +56,9 @@ fun BitwardenActionCard(
         elevation = CardDefaults.elevatedCardElevation(),
         border = BorderStroke(width = 1.dp, color = BitwardenTheme.colorScheme.stroke.border),
     ) {
-        Column(
-            modifier = Modifier.padding(16.dp),
-        ) {
-            Row(
-                modifier = Modifier.fillMaxWidth(),
-            ) {
+        Row(modifier = Modifier.fillMaxWidth()) {
+            Spacer(Modifier.width(16.dp))
+            Row(modifier = Modifier.padding(top = 16.dp)) {
```

Comment:
💭 non-blocking but the use of the top padding here but the trailing spacer at the end of the card content seems disjointed. I am an expert in disjointed 😆 😢

Reply:
I agree and I was not super happy with how this turned out.

But I needed the close icon to not be padded while making sure everything else was.